test/helpers/cmd.go

```diff
@@ -279,6 +279,33 @@ func (res *CmdRes) WaitUntilMatchRegexp(expr string) error {
 		&TimeoutConfig{Timeout: HelperTimeout})
 }
 
+// FilterBuffer structs that extends buffer methods
```

Comment:
Small nit; can you put this type and the functions that have it as a receiver in a separate file?